### DISCUSSION TITLE
Can standard parameters set of FIPS204 (AKA Dilithium) launched on August of 2024 be perfectly aggregated threshold?

### DISCUSSION THREAD

**Discussion by r8raq** [paulmillr/noble-post-quantum#11] [General]
To be aggregated after threshold of Dilithium via its standard parameters set launched on August of 2024 via NISTPQC, is it easy to get anything done and implementation perfectly on current bitcoin, or Ethereum?

What're the potential trade-offs or flaw points?any thoughts shared？Paul！

**Comment by paulmillr** [comment]
Do you want to switch Bitcoin to Dilithium?

**Comment by tomekmarchi** [comment]
> To be aggregated after threshold of Dilithium via its standard parameters set launched on August of 2024 via NISTPQC, is it easy to get anything done and implementation perfectly on current bitcoin, or Ethereum?
> 
> What're the potential trade-offs or flaw points?any thoughts shared？Paul！

Put it this way BTC has much bigger architectural issues and concerns before it is looking at a PQ sig scheme swap. BTC's core code isn't in JS to begin with and the pure JS version should not be used for performance critical applications.

@paulmillr let's close this spam.
